src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let numberOfCalls = 0;
+
+  return function add(firstNumber) {
+    return function anotherAdd(secondNumber) {
```

Comment:
For simplicity, you could return anonymous functions (naming is usually hard part of coding) :)
```suggestion
  return function(firstNumber) {
    return function(secondNumber) {
```